docs: ENSv2-focused refinements (#2099)
namehash/ensnode: docs/ensnode.io/src/content/docs/docs/services/ensapi/index.mdx

import { LinkCard } from '@astrojs/starlight/components';

ENSApi is a web service built on top of [ENSDb](/docs/services/ensdb), and provides access to indexed ENS data through multiple HTTP APIs.

## GraphQL APIs

### ENS Omnigraph API

The **Omnigraph API** is the _new_ primary GraphQL API for ENSNode that provides a unified access model for both ENSv1 and ENSv2 data. Write your queries once and get automatic support for ENSv1 and ENSv2 data!

### ENS Subgraph API

> The Subgraph API is the _legacy_ GraphQL API for ENSNode that was built to provide a drop-in replacement for The Graph's ENS subgraph. It is now deprecated in favor of the Omnigraph API, but is still available for compatibility reasons.

Reviewer: ```suggestion
The Subgraph API is the **deprecated** _legacy_ ENSv1-only GraphQL API. It is now deprecated in favor of the Omnigraph API, but is still available for backwards compatibility reasons with The Graph's ENS subgraph.
```

## REST APIs

> ENSApi also provides a set of REST APIs for accessing ENS data. These APIs are designed to be simple and easy to use, and are ideal for use cases where GraphQL is not a good fit.

Reviewer: ```suggestion
A growing set of REST APIs are also being added to ENSApi.
```

<LinkCard
  title="ENSApi Reference"

>   description="Learn more about REST APIs from ENSApi Reference"

Reviewer: ```suggestion
  description="Learn more about ENSApi's REST APIs"
```
